release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1003,6 +1027,15 @@ Starting with {product-title} 4.6, {op-system-first} switched to using `NetworkM
 
 Cluster Loader is now deprecated and will be removed in a future release.
 
+[id="ocp-4-8-builds-lasttriggeredimageid-parameter"]
+==== The lastTriggeredImageID parameter in builds is deprecated
+
+This release deprecates the `lastTriggeredImageID` in the `ImageChangeTrigger` object, which is one of the `BuildTriggerPolicy` types that can be set on a `BuildConfig` spec.
+
+{product-title} version 4.9 will remove support for `lastTriggeredImageID` and ignore it. Then, image change triggers will not start a build based on changes to the `lastTriggeredImageID` field in the `BuildConfig` spec. Instead, the image IDs that trigger a build will be recorded in the status of the `BuildConfig` object, which most users cannot change.
+
+Therefore, update scripts and jobs that inspect `checkingbuildConfig.spec.triggers[i].imageChange.lastTriggeredImageID` accordingly. (link:https://issues.redhat.com/browse/BUILD-213[*BUILD-213*])
```

Comment:
`checkingbuildConfig.spec.triggers[i].imageChange.lastTriggeredImageID` is better to be `buildConfig.spec.triggers[i].imageChange.lastTriggeredImageID`

Reply:
Thanks. Updated.